src/server/api/trpc.ts

```diff
@@ -20,7 +20,7 @@ import { type CreateNextContextOptions } from "@trpc/server/adapters/next";
 import { type Session } from "next-auth";
 
 import { getServerAuthSession } from "../auth";
-import { prisma } from "../db";
+import { prisma, smartsheet } from "../db";
```

Comment:
I think smartsheet probably shouldn't come from the db file, but that's a nitpick

Reply:
Once we transfer everything to db, this will be removed anyways